### PR TITLE
VS/Add a test to check if password can be unmasked when having a PP

### PR DESCRIPTION
**Pull request by vsangereanMOZ**
### Description
Add a test to check if password can be unmasked when having a Primary Password

### Bugzilla bug ID

**Testrail:** https://mozilla.testrail.io/index.php?/cases/view/2264688
**Link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1920196

### Type of change

Please delete options that are not relevant.

- [x] New Test
- [ ] New POM
- [ ] Other Changes (Please specify)

### How does this resolve / make progress on that bug?

Please describe the progress or significance with respect to the bug listed above.

### Screenshots / Explanations

Please upload any relevant media or add a relevant description with respect to the bug listed above.

### Comments / Concerns

Please add a short blurb about any comments or concerns that this change might cause.

### CODE REVIEW

File: tests/password_manager/test_can_view_passoword_when_PP_enabled.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+        primary_password_input_field = about_logins.get_element(
+            "primary-password-dialog-input-field"
+        )
+        primary_password_input_field.send_keys("securePassword1")
+        primary_password_input_field.send_keys(Keys.ENTER)
+
+    # Verify that the password is unmasked by checking that the type is now text.
+    driver.switch_to.window(driver.window_handles[0])
+    password_field = about_logins.get_element("about-logins-page-password-field")
```
Comment (ben-c-at-moz) [inline tests/password_manager/test_can_view_passoword_when_PP_enabled.py:83]: I might replace this line and line 84 with one line:
`about_logins.element_attribute_contains("about-logins-page-password-field", "type", "text")`

File: tests/password_manager/test_can_view_passoword_when_PP_enabled.py
Comment (ben-c-at-moz) [inline tests/password_manager/test_can_view_passoword_when_PP_enabled.py:1]: Typo in filename: passoword should be password

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]
A couple of quick fixes, but it's a good test.

File: tests/password_manager/test_can_view_passoword_when_PP_enabled.py (hunk 1 above)
Comment (vsangereanMOZ) [reply tests/password_manager/test_can_view_passoword_when_PP_enabled.py]: Unfortunately this does not work. It fails with a timeout exception.

File: tests/password_manager/test_can_view_passoword_when_PP_enabled.py
Comment (vsangereanMOZ) [reply tests/password_manager/test_can_view_passoword_when_PP_enabled.py:1]: done

**Review by ben-c-at-moz — APPROVED** [review]
I did a couple of fixes, should be fine. Merging as VS is on sick day

**Pull request merged**